addon.xml

```diff
@@ -5,6 +5,9 @@
         provider-name="Jellyfin Contributors, angelblue05">
   <requires>
     <import addon="xbmc.python" version="2.25.0"/>
+    <import addon="script.module.requests" version="2.22.1"/>
+    <import addon="script.module.dateutil" version="2.8.0"/>
```

Comment:
```suggestion
    <import addon="script.module.dateutil" version="2.7.3"/>
```
Maybe this is why the full source code was in the repo instead of importing it, but the latest version of the library available for kodi is 2.7.3.  https://github.com/Razzeee/script.module.dateutil
As far as I can tell, going down to 2.7.3 doesn't hurt anything.  I was able to sync my library and playback files with the older version just fine.  We could also request upstream update to a newer version